pages/index.tsx

```diff
@@ -50,6 +51,7 @@ export default function Home() {
                                 <p>Don't bother with other chat apps.</p>
                                 <h1>Communication is critical.<br />Privacy is essential.</h1>
                                 <JoinLinks />
+                                <h3>(<Link href="/desktop">See here</Link> for more information on downloading Revolt Desktop.)</h3>
```

Comment:
```suggestion
                                <h3><Link href="/desktop">See here</Link> for more information on downloading Revolt Desktop.</h3>
```
Remove the parentheses, text looks awkward, you should probably even consider finding a different color or text size for it.

Reply:
I feel like it looks a bit awkward *without* the parentheses, but I'd like to hear a few more thoughts. I'll try and colour the link.